engine/access/rpc/backend/backend_accounts.go

```diff
@@ -14,43 +15,49 @@ import (
 	"github.com/onflow/flow-go/engine/common/rpc"
 	"github.com/onflow/flow-go/engine/common/rpc/convert"
 	"github.com/onflow/flow-go/model/flow"
+	"github.com/onflow/flow-go/module/execution"
 	"github.com/onflow/flow-go/state/protocol"
 	"github.com/onflow/flow-go/storage"
 )
 
 type backendAccounts struct {
+	log               zerolog.Logger
 	state             protocol.State
 	headers           storage.Headers
 	executionReceipts storage.ExecutionReceipts
 	connFactory       connection.ConnectionFactory
-	log               zerolog.Logger
 	nodeCommunicator  Communicator
+	scriptExecutor    execution.ScriptExecutor
+	scriptExecMode    ScriptExecutionMode
 }
 
+// GetAccount returns the account details at the latest sealed block.
+// Alias for GetAccountAtLatestBlock
 func (b *backendAccounts) GetAccount(ctx context.Context, address flow.Address) (*flow.Account, error) {
 	return b.GetAccountAtLatestBlock(ctx, address)
 }
 
+// GetAccountAtLatestBlock returns the account details at the latest sealed block.
 func (b *backendAccounts) GetAccountAtLatestBlock(ctx context.Context, address flow.Address) (*flow.Account, error) {
-
 	// get the latest sealed header
-	latestHeader, err := b.state.Sealed().Head()
+	sealed, err := b.state.Sealed().Head()
 	if err != nil {
 		return nil, status.Errorf(codes.Internal, "failed to get latest sealed header: %v", err)
 	}
 
 	// get the block id of the latest sealed header
-	latestBlockID := latestHeader.ID()
+	sealedBlockID := sealed.ID()
 
-	account, err := b.getAccountAtBlockID(ctx, address, latestBlockID)
+	account, err := b.getAccountAtBlock(ctx, address, sealedBlockID, sealed.Height)
 	if err != nil {
-		b.log.Error().Err(err).Msgf("failed to get account at blockID: %v", latestBlockID)
+		b.log.Debug().Err(err).Msgf("failed to get account at blockID: %v", sealedBlockID)
```

Comment:
this could fail for a lot of reasons that aren't errors occurring on the node, so I figure it's better to produce fewer error level messages in that case.